ontologies/ebucore.rdfs.ttl

```diff
@@ -163,3 +174,47 @@ ebucore:hasCastMember a rdf:Property ;
   rdfs:comment "A member of the cast."@en, "Een lid van de cast."@nl, "A member of the cast."@fr, "Range: a string or a Cast"@en ;
   rdfs:range xsd:string, ebucore:Cast ;
   rdfs:domain ebucore:Asset .
+
+ebucore:regionDelimX a rdf:Property ;
+  rdfs:domain ebucore:MediaFragment ;
+  rdfs:range xsd:string, xsd:anyURI ;
+  rdfs:subPropertyOf ebucore:hasAnnotationPurpose ;
+  rdfs:label "Region delimiter ()"@en ;
+  rdfs:label "Regio-afbakening (x-as)"@fr ;
+  rdfs:label "Délimiteur de région (axe des x)"@nl ;
+  skos:definition "To define the top left corner of a zone on the x-axis."@en ;
+  skos:definition "Définir le coin supérieur gauche d'une zone sur l'axe des x."@fr ;
+  skos:definition "Het definiëren van de linkerbovenhoek van een zone op de x-as."@nl .
+
+ebucore:regionDelimY a rdf:Property ;
+  rdfs:domain ebucore:MediaFragment ;
+  rdfs:range xsd:string, xsd:anyURI ;
+  rdfs:subPropertyOf ebucore:hasAnnotationPurpose ;
+  rdfs:label "Region delimiter (y-axis)"@en ;
+  rdfs:label "Regio-afbakening (y-as)"@fr ;
+  rdfs:label "Délimiteur de région (axe des y)"@nl ;
+  skos:definition "To define the top left corner of a zone on the y-axis."@en ;
+  skos:definition "Définir le coin supérieur gauche d'une zone sur l'axe des y."@fr ;
+  skos:definition "Het definiëren van de linkerbovenhoek van een zone op de y-as."@nl .
+
+ebucore:width a rdf:Property ;
+  rdfs:domain ebucore:MediaFragment ;
+  rdfs:range xsd:string, xsd:anyURI ;
+  rdfs:subPropertyOf ebucore:hasAnnotationPurpose ;
+  rdfs:label "Width"@en ;
```

Comment:
lowercase? (or are other ebucore properties also capitalized?)

Reply:
It is a mix of both, so I will all put them in lowercase